### PR TITLE
feat(suite-native): no portfolio title for device connection enabled

### LINKED ISSUE #12709: Remove "my portfolio balance headline"
Remove the headline "My portfolio balance" from connected trezor area. 

With introducing passphrase and view-only where user can have more devices in view-only and passphrase it is confusing for users, what does it mean as it can make impression that it is a total of everything they have on all wallets / devices

### PR DESCRIPTION
**Pull request by vytick**
Remove "My Portfolio Balance" from graph

## Related Issue

Resolve #12709

### CODE REVIEW

File: suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraphHeader.tsx [hunk 1]
```diff
@@ -57,14 +58,17 @@ export const PortfolioGraphHeader = () => {
...
+    const [isUsbDeviceConnectFeatureEnabled] = useFeatureFlag(FeatureFlag.IsDeviceConnectEnabled);
     const isLoading = isDiscoveryActive || !isDeviceAuthorized;
 
     return (
         <Box>
             <VStack spacing="extraSmall" alignItems="center">
-                <Text color="textSubdued" variant="hint">
-                    <Translation id="moduleHome.graph.title" />
-                </Text>
+                {!isUsbDeviceConnectFeatureEnabled && (
```
Comment (juriczech) [inline suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraphHeader.tsx]: I think this won't be correct for:
- usb connection enabled and selected device is portfolio tracker

Comment (juriczech) [reply suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraphHeader.tsx]: Altough, after reading the task description, it's not clear what's correct behaviour to me. Could you please double-check with @shenkys?

Comment (vytick) [reply suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraphHeader.tsx]: Hmm. Yah, it can be read two ways. i'll check, good catch.

Comment (vytick) [reply suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraphHeader.tsx]: I've clarified with @shenkys  and the result is: Remove that title for good.

Comment (vytick) [reply suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraphHeader.tsx]: rebased and changed

Comment (vytick) [reply suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraphHeader.tsx]: [b2a992d](https://github.com/trezor/trezor-suite/pull/13070/commits/b2a992d41fdfbdf3a90ac933bfc93466942a9ffa)

**Review by juriczech — APPROVED** [review]
LGTM, haven't tested it. Tests are failing though.

**Pull request merged**